understanding/21/non-text-contrast.html

```diff
@@ -200,10 +200,14 @@ <h4>User Interface Component Examples</h4>
 						<th>Examples</th>
 						</tr>
 						<tr>
-							<th>Link Text</th>
+							<th>Default link text</th>
 							<td>Default link text is in the scope of <a href="contrast-minimum">1.4.3 Contrast (Minimum)</a>, and the underline is sufficient to indicate the link.</td>
 							<td><img src="img/link-text-default.png" alt="A browser-default styled link, blue with an underline."/></td>
 						</tr>
+						<th>Colored underline is the only indicator of a link - fail</th>
```

Comment:
missing an opening `<tr>`

Reply:
the table is prefaced with "the following are examples that have sufficient contrast" so this fail example is the odd one out now

Reply:
Oh interesting. I didn't spot that text. The table also has "Checkbox - Fail" and "Checkbox - Subtle focus style - fail", so we could either yoink out that "sufficient contrast" sentence or re-do the existing two fails as well.